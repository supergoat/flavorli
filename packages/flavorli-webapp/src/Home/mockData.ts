// export {recipes} from '../__mockData__';

export const recipes = [];
