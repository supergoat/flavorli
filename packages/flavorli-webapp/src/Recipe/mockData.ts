export {recipes} from '../__mockData__';
