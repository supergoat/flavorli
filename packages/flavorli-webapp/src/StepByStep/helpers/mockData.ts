import {IStep} from '../types';

export const steps: IStep[] = [
  {
    no: 1,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description: 'Place a pot of water in medium heat and set aside to boil',
    ingredients: [],
    kitchenware: [
      {
        qty: '1',
        name: 'Pot',
      },
    ],
  },
  {
    no: 2,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description: 'Place a sauce pan over high heat and add the olive oil',
    ingredients: [
      {
        qty: '4 - 6 tbsps',
        name: 'olive oil',
      },
    ],
    kitchenware: [
      {
        qty: '1',
        name: 'Sauce Pan',
      },
    ],
  },
  {
    no: 3,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description:
      'Cut the eggplan into 1 cm cubes. Add it to the pan and sauté for 1-2 minutes',
    ingredients: [
      {
        qty: '2',
        name: 'eggplants',
      },
    ],
    kitchenware: [
      {qty: '1', name: 'Knife'},
      {qty: '1', name: 'Cutting Board'},
      {qty: '1', name: 'Spoon'},
    ],
  },
  {
    no: 4,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description:
      'Finely chop the onion, add it to the pan and sauté for 1-2 minutes',
    ingredients: [
      {
        qty: '1',
        name: 'onion',
      },
    ],
    kitchenware: [],
  },
  {
    no: 5,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description:
      'Cut the zucchinis and the peppers into small cubes and add them to the pan. Sauté for 4-5 minutes, until the vegetables are ready',
    ingredients: [
      {
        qty: '2',
        name: 'zucchinis',
      },
      {
        qty: '1',
        name: 'red bell pepper',
      },
      {
        qty: '1',
        name: 'green bell pepper',
      },
    ],
    kitchenware: [],
  },
  {
    no: 6,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description:
      'Add the salt, pepper, oregano, thyme, chili flakes, sugar and tomato paste and sauté',
    ingredients: [
      {
        qty: '',
        name: 'salt',
      },
      {
        qty: '',
        name: 'pepper',
      },
      {
        qty: '1 tsp',
        name: 'oregano',
      },
      {
        qty: '1 tbsp',
        name: 'thyme',
      },
      {
        qty: '1 tsp',
        name: 'chilli flakes',
      },
      {
        qty: '1 tbsp',
        name: 'granulated sugar',
      },
      {
        qty: '1 tsp',
        name: 'tomato paste',
      },
    ],
    kitchenware: [
      {
        qty: '1',
        name: 'Tea Spoon',
      },
    ],
  },
  {
    no: 7,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [],
    description:
      'Add the wine and canned tomatoes and simmer at medium-low heat for 10 minutes',
    ingredients: [
      {
        qty: '40 g',
        name: 'white wine',
      },
      {
        qty: '400 g',
        name: 'canned tomatoes',
      },
    ],
    timer: {id: 7, name: 'Step 7: Simmer Sauce', minutes: 10, seconds: 0},
    kitchenware: [{qty: '1', name: 'scale'}],
  },
  {
    no: 8,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [{from: 1, heading: 'From Step 1', name: 'Boiling water'}],
    description:
      'In the pot with boiling water, add salt along with the macaroni, and boil for ten minutes',
    ingredients: [
      {
        qty: '',
        name: 'salt',
      },
      {
        qty: '500 g',
        name: 'macaroni',
      },
    ],
    timer: {id: 8, name: 'Step 8: Cook Pasta', minutes: 10, seconds: 0},
    kitchenware: [],
  },
  {
    no: 9,
    tag: {
      text: 'Assembling',
      color: 'tagPink',
    },
    links: [],
    description: 'Preheat the oven at 180 C set to fan',
    ingredients: [],
    kitchenware: [],
  },
  {
    no: 10,
    tag: {
      text: 'Bechamel',
      color: 'tagOrange',
    },
    links: [],
    description:
      'Place a pot over medium heat. Add the olive oil and the flour and using a hand whisk, mix well until all of the flour is absorbed',
    ingredients: [
      {
        qty: '80 g',
        name: 'olive oil',
      },
      {
        qty: '80 g',
        name: 'all-purpose flour',
      },
    ],
    kitchenware: [
      {qty: '1', name: 'Small Pot'},
      {qty: '1', name: 'Hand Whisk'},
    ],
  },
  {
    no: 11,
    tag: {
      text: 'Bechamel',
      color: 'tagOrange',
    },
    links: [],
    description: 'Add the soy milk in 7-8 batches, stirring constantly',
    ingredients: [
      {
        qty: '800 g',
        name: 'soy milk',
      },
    ],
    kitchenware: [],
  },
  {
    no: 12,
    tag: {
      text: 'Bechamel',
      color: 'tagOrange',
    },
    links: [],
    description:
      'Season with salt and pepper and as soon as it thickens and comes to boil, remove from the heat',
    ingredients: [
      {
        qty: '',
        name: 'salt',
      },
      {
        qty: '',
        name: 'pepper',
      },
    ],
    kitchenware: [],
  },
  {
    no: 13,
    tag: {
      text: 'Sauce',
      color: 'tagRed',
    },
    links: [
      {
        from: 7,
        heading: 'From Step 7',
        name: 'Pan with vegetables',
        timerId: 7,
      },
      {
        from: 8,
        heading: 'From Step 8',
        name: 'Macaroni',
        timerId: 8,
      },
    ],
    description:
      'Drain the macaroni and transfer to the pan with the vegetables. Mix and set aside',
    ingredients: [],
    kitchenware: [{qty: '1', name: 'Colander'}],
  },
  {
    no: 14,
    tag: {
      text: 'Assembling',
      color: 'tagPink',
    },
    links: [{from: 12, heading: 'From Step 12', name: 'Bechamel sauce'}],
    description:
      'Add 100g of the bechamel into the pasta mixture, add oregano, mint, and mix well',
    ingredients: [
      {
        qty: '1 tsp',
        name: 'oregano',
      },
      {
        qty: '1 tbsp',
        name: 'mint',
      },
    ],
    kitchenware: [],
  },
  {
    no: 15,
    tag: {
      text: 'Assembling',
      color: 'tagPink',
    },
    links: [],
    description:
      'Grease 40 cm ovenproof baking dish with olive oil and sprinke the breadcrumps',
    ingredients: [
      {
        qty: '1 tbsp',
        name: 'olive oil',
      },
      {
        qty: '25 g',
        name: 'breadcrumps',
      },
    ],
    kitchenware: [{qty: '1', name: '40 cm ovenproof baking dish'}],
  },
  {
    no: 16,
    tag: {
      text: 'Assembling',
      color: 'tagPink',
    },
    links: [],
    description:
      'Add the pasta mixture, spread the bechamel sauce, and sprinke the breadcrumps, olive oil, thyme and pepper',
    ingredients: [
      {
        qty: '25 g',
        name: 'breadcrumps',
      },
      {
        qty: '1 tbsp',
        name: 'olive oil',
      },
      {
        qty: '1 tbsp',
        name: 'thyme',
      },
      {
        qty: '',
        name: 'pepper',
      },
    ],
    kitchenware: [],
  },
  {
    no: 17,
    tag: {
      text: 'Assembling',
      color: 'tagPink',
    },
    links: [],
    description: 'Bake for 25 - 30 minutes. Remove and let it cool',
    ingredients: [],
    timer: {id: 17, name: 'Step 17: Bake Pastitsio', minutes: 30, seconds: 0},
    kitchenware: [
      {
        qty: '1',
        name: 'Oven Mitts',
      },
    ],
  },
];
