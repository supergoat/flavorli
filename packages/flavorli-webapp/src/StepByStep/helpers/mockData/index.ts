// https://www.youtube.com/watch?v=q1q3HfVvzQ0
import {ingredients} from './ingredients';
import {items} from './items';
import {preparationSteps} from './preparationSteps';
import {recipeSteps} from './recipeSteps';
import {IStepByStep} from '../../types';

export const stepByStep: IStepByStep = {
  intro: {
    name: 'Greek Pastitsio',
    image:
      'https://akispetretzikis.com/system/uploads/medium/data/14955/recipe_main_vegan-pastitsio.jpg',
    preparation: '25 minutes',
    cooking: '40 minutes',
    portions: '8 - 10',
    difficulty: 'Level 2',
  },
  ingredients,
  items,
  preparationSteps,
  recipeSteps,
};

export {ingredients, items, preparationSteps, recipeSteps};
