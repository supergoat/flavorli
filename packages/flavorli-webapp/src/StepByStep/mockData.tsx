import {IStep, IRecipeTimer, ILink, IRecipe} from '../types';

export const recipes: IRecipe[] = [
  {
    id: '1',
    createdAt: '2020-01-19T19:37:42.873Z',
    author: 'Robin Robertson',
    name: 'Cherry Berry Smoothie',
    image:
      'https://flavorli-bucket-2.s3.eu-west-2.amazonaws.com/CherryBerrySmoothie.jpeg',
    preparationTime: 7,
    cookingTime: 0,
    portions: '1',
    difficulty: '1',
    ingredients: [
      {
        qty: '1 cup/100 g',
        name: 'Frozen Blueberries',
      },
      {
        qty: '1/2 cup/110 g',
        name: 'fresh or frozen pitted cherries',
      },
      {
        qty: '1',
        name: 'frozen ripe banana, cut into chunks before freezing',
      },
      {
        qty: '1 tbsp',
        name: 'ground flaxseeds (or linseeds)',
      },
      {
        qty: '1 tbsp',
        name: 'almond butter (or peanut butter)',
      },
      {
        qty: '1 cup/22.5 g',
        name: 'raw spinach leaves',
      },
      {
        qty: '1 1/2 cup (360 ml)',
        name: 'water',
      },
    ],
    items: [
      {
        qty: '1',
        name: 'Blender',
      },
      {
        qty: '1',
        name: 'Cup',
      },
      {
        qty: '1/2',
        name: 'cup',
      },
      {
        qty: '1',
        name: 'tablespoon',
      },
    ],
    steps: [
      {
        no: 1,
        type: 'MISE_EN_PLACE',
        tag: {
          text: 'Smoothie',
          color: 'tagRed',
        },
        items: [
          {
            qty: '1',
            name: 'Blender',
          },
        ],
        ingredients: [
          {
            qty: '1 cup/100 g',
            name: 'Frozen Blueberries',
          },
          {
            qty: '1/2 cup/110 g',
            name: 'fresh or frozen pitted cherries',
          },
          {
            qty: '1',
            name: 'frozen ripe banana, cut into chunks before freezing',
          },
          {
            qty: '1 tbsp',
            name: 'ground flaxseeds (or linseeds)',
          },
          {
            qty: '1 tbsp',
            name: 'almond butter (or peanut butter)',
          },
          {
            qty: '1 cup/22.5 g',
            name: 'raw spinach leaves',
          },
          {
            qty: '1 1/2 cup (360 ml)',
            name: 'water',
          },
        ],
        tasks: [
          'Add 1 cup/100 g Frozen Blueberries in a blender',
          'Add 1/2 cup/110 g fresh or frozen pitted cherries',
          'Add 1 frozen ripe banana, cut into chunks before freezing',
          'Add 1 tbsp ground flaxseeds (or linseeds)',
          'Add 1 tbsp almond butter (or peanut butter)',
          'Add 1 cup/22.5 g raw spinach leaves',
          'Add 1 1/2 cup cup (360 ml) water',
        ],
        images: [],
      },
      {
        no: 2,
        type: 'PREPARATION',
        tag: {
          text: 'Smoothie',
          color: 'tagRed',
        },
        items: [],
        ingredients: [],
        tasks: [
          'Blend all the ingredients until smooth and creamy for about 1 minute',
          'Add more water if you prefer a thinner consistency',
          'Serve immediately',
        ],
        timer: {
          id: '2',
          name: 'Step 2: Blend',
          minutes: 1,
          seconds: 0,
        },
        images: [],
      },
    ],
  },
];

export const timer: IRecipeTimer = {
  id: '1',
  name: 'Thicken Sauce',
  minutes: 10,
  seconds: 0,
  isPaused: true,
  updatedAt: '2020-01-26T15:15:06.689Z',
  remainingTime: 600000,
};

export const link: ILink = {
  from: 1,
  heading: 'From Step 1',
  name: 'Thicken Sauce',
};

export const linkWithTimer: ILink = {
  from: 1,
  heading: 'From Step 1',
  name: 'Thicken Sauce',
  timerId: '1',
};

export const items = [
  {
    qty: '1',
    name: 'Pot',
  },
];

export const preparationStep: IStep = {
  type: 'MISE_EN_PLACE',
  tag: {
    text: 'Sauce',
    color: 'tagRed',
  },
  tasks: ['Add the coconul oil in a ramekin and place it first in order'],
  ingredients: [
    {
      qty: '2 tbsps',
      name: 'coconut oil',
    },
  ],
  items: [
    {
      qty: '1',
      name: 'Ramekin',
    },
  ],
  images: [],
};

export const recipeStep: IStep = {
  type: 'PREPARATION',
  no: 1,
  tag: {
    text: 'Sauce',
    color: 'tagRed',
  },
  links: [],
  tasks: ['Heat the coconut oil in a small pot on low heat'],
  ingredients: [
    {
      qty: '2 tbsps',
      name: 'coconut oil',
    },
  ],
  items: [
    {
      qty: '1',
      name: 'Small Pot',
    },
  ],
  images: [],
};

export const recipeSteps = [recipeStep];

export const introStep = {
  author: 'Akis Petratzikis',
  name: 'Greek Pastitsio',
  image: '',
  preparation: '25 minutes',
  cooking: '40 minutes',
  portions: '8 - 10',
  difficulty: 'Level 2',
};

export const ingredients = [{qty: '1', name: 'Onion'}];

export const images = [
  {
    src: '',
    alt: 'image 1',
  },

  {
    src: '',
    alt: 'image 2',
  },
];
