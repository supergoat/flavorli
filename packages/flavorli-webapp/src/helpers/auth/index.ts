import {CognitoUserPool} from 'amazon-cognito-identity-js';
import * as AWS from 'aws-sdk/global';
import {authenticate} from './authenticate';
import {createNewUser} from './createNewUser';
import {establishUserSession} from './establishUserSession';
import {verifyPhoneNumber} from './verifyPhoneNumber';

AWS.config.region = 'eu-west-2';

export const userPool = new CognitoUserPool({
  UserPoolId: process.env.REACT_APP_COGNITO_USER_POOL_ID as string,
  ClientId: process.env.REACT_APP_COGNITO_CLIENT_ID as string,
});

export const loginId = `cognito-idp.${
  AWS.config.region
}.amazonaws.com/${userPool.getUserPoolId()}`;

export {authenticate, createNewUser, establishUserSession, verifyPhoneNumber};
