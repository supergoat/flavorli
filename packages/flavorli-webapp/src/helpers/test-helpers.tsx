import React from 'react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import {ThemeProvider} from 'styled-components';
import {render as testRender} from '@testing-library/react';
import {theme} from '@flavorli/elements';

export function render(ui: React.ReactNode) {
  const utils = testRender(<ThemeProvider theme={theme}>{ui}</ThemeProvider>);

  return utils;
}

export function renderWithRouter(ui: React.ReactElement, route: string = '/') {
  const history = createMemoryHistory({initialEntries: [route]});
  const Wrapper = ({children}: {children: React.ReactNode}) => (
    <Router history={history}>{children}</Router>
  );
  return {
    ...testRender(<ThemeProvider theme={theme}>{ui}</ThemeProvider>, {
      wrapper: Wrapper as React.ComponentType,
    }),
    // adding `history` to the returned utilities to allow us
    // to reference it in our tests (just try to avoid using
    // this to test implementation details).
    history,
  };
}
