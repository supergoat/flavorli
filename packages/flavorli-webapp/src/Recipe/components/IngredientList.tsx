import React from 'react';
import {Stack, H3} from '@flavorli/elements';
import {IIngredient} from '../../StepByStep/types';
import Ingredient from './Ingredient';

interface IIngredientListProps {
  taskName: string;
  ingredients: IIngredient[];
}
const IngredientList = ({taskName, ingredients}: IIngredientListProps) => {
  return (
    <Stack gap={8} alignment="center" width="100%">
      <H3 width="100%">{taskName}</H3>
      {ingredients.map((ingredient, index2) => {
        return <Ingredient ingredient={ingredient} key={ingredient.name} />;
      })}
    </Stack>
  );
};

export default IngredientList;
