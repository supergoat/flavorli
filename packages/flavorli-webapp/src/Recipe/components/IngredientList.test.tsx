import React from 'react';
import {render} from '../../helpers/test-helpers';
import {axe} from 'jest-axe';
import IngredientList from './IngredientList';
import {recipe} from '../mockData';

const setup = () => {
  return render(
    <IngredientList
      taskName={recipe.tasks[0].name}
      ingredients={recipe.tasks[0].ingredients}
    />,
  );
};

describe('IngredientList', () => {
  it('should not have any axe violations', async () => {
    const {container} = setup();
    const results = await axe(container);
    expect(results).toHaveNoViolations();
  });

  it('should render correctly', () => {
    const {container} = setup();
    expect(container.firstChild).toMatchSnapshot();
  });
});
