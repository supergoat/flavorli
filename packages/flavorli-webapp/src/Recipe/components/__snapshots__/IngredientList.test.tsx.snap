// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IngredientList should render correctly 1`] = `
<div>
  <div
    class="sc-AykKH bFnGQL"
    direction="vertical"
    width="100%"
  >
    <h3
      class="sc-AykKF lfwwnG"
      width="100%"
    >
      Sauce
    </h3>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        2
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        eggplants
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        4-6 tbsps
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        olive oil
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        onion
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1 clove
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        garlic
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        2
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        zucchinis
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        red bell pepper
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        green bell pepper
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1 tsp
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        oregano
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1 tbsp
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        thyme
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1 tsp
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        chilli flakes
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1 tbsp
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        granulated sugar
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        1 tbsp
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        tomato paste
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        50 g
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        white wine
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        400 g
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        canned tomatoes
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      >
        500 g
      </p>
      <p
        class="sc-AykKG fCXClx"
      >
        macaroni
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      />
      <p
        class="sc-AykKG fCXClx"
      >
        salt
      </p>
    </div>
    <div
      class="sc-AykKH dujSYy"
      direction="horizontal"
      width="100%"
    >
      <p
        class="sc-AykKG iIcCYk"
        width="25%"
      />
      <p
        class="sc-AykKG fCXClx"
      >
        pepper
      </p>
    </div>
  </div>
</div>
`;
